doc/software.rst

```diff
@@ -16,14 +16,14 @@ If ``pip`` is unable to automatically install its dependencies you will have to
 add them manually.  To resolve the issues you will need to add the appropriate
 development packages to continue.
 
-If you are using Raspbian Stretch or Raspberry Pi OS (Buster) you should
+If you are using Raspbian Stretch or Raspberry Pi OS you should
```

Comment:
If you are using ~~Raspbian Stretch or~~ Raspberry Pi OS you should

Reply:
Stretch is still called Raspbian but it's confusing, I'll fix.